mobilecoind/src/service.rs

```diff
@@ -2103,7 +2112,9 @@ build_api! {
 
     // Database encryption
     set_db_password SetDbPasswordRequest Empty set_db_password_impl,
-    unlock_db UnlockDbRequest Empty unlock_db_impl
+    unlock_db UnlockDbRequest Empty unlock_db_impl,
+
+    get_version Empty MobilecoindVersionResponse get_version_impl
```

Comment:
IIUC we can add a trailing comma so future additions don't need to modify this line.

```suggestion
    get_version Empty MobilecoindVersionResponse get_version_impl,
```